packages/devtools_app/test/analytics_prompt_test.dart

```diff
@@ -32,6 +33,7 @@ void main() {
     setUp(() {
       didCallEnableAnalytics = false;
       setGlobal(ServiceConnectionManager, FakeServiceManager());
+      setGlobal(IdeTheme, IdeTheme());
```

Comment:
do all these test fail if the IdeTheme global is null?

Reply:
yes, they fail

Reply:
Would it be better to make sure that we fail gracefully at runtime when IdeTheme is null? If the field is nullable in general, we are saying it is okay to have a null ide theme at runtime, so we want to make sure that our UI gracefully handles the null value and does not throw exceptions.

Reply:
Instead of investing in graceful error handling, I would invest in making sure getIdeTheme(), invoked in main.dart, never returns null.

Reply:
updated ide_theme_stub.dart